src/RFC-0122_Burning.md

```diff
@@ -0,0 +1,180 @@
+For each burned output, we need to have a kernel where the burned output's commitment is stored. 
+
+```rust,ignore
+pub struct TransactionKernel {
+    pub version: TransactionKernelVersion,
+    /// Options for a kernel's structure or use
+    pub features: KernelFeatures,
+    /// Fee originally included in the transaction this proof is for.
+    pub fee: MicroTari,
+    /// This kernel is not valid earlier than lock_height blocks
+    /// The max lock_height of all *inputs* to this transaction
+    pub lock_height: u64,
+    /// Remainder of the sum of all transaction commitments (minus an offset). If the transaction is well-formed,
+    /// amounts plus fee will sum to zero, and the excess is hence a valid public key.
+    pub excess: Commitment,
+    /// An aggregated signature of the metadata in this kernel, signed by the individual excess values and the offset
+    /// excess of the sender.
+    pub excess_sig: Signature,
+    /// This is an optional field that must be set if the transaction contains a burned output.
+    pub burn_commitment: Option<Commitment>,
+
+}
+```
+Each kernel also has a field called [KernelFeatures](https://github.com/tari-project/tari/blob/2ca06724f0ab7c63eb0b6caab563372f353f4348/base_layer/core/src/transactions/transaction_components/kernel_features.rs#L36) that unique properties of the kernel. These fields need to be expanded to include a burn type.
+Inside this field, we track every possible type of output. We need to add another type here called `BURNED_KERNEL`. 
+
+```rust,ignore
+pub struct KernelFeatures: u8 {
+        /// Coinbase transaction
+        const COINBASE_KERNEL = 1u8;
+        /// Burned output
+        const BURNED_KERNEL = 2u8;
+    }
+```
+
+For each burned output in a transaction, there needs to be a kernel. This means that if a transaction has two burned outputs, it needs at least two kernels. 
+This stops a node from publishing an aggregated inflated commitment in the kernel. 
+
+
+
+### Consensus rules changes
+
+When a block or transaction is received with a burned output, there:
+* MUST be a kernel containing a commitment matching exactly one and only one output that is flagged as burned.
+
+### Chain balance equation changes 
+
+When checking the total chain emission, the following equation must hold:
+$$
+\begin{align}
+&\sum_i\mathrm{C_{i}} \stackrel{?}{=} E_l \cdot H + \sum_k\mathrm{K_k} + \text{offset}\\\\
+& \text{for each unspent output}, i, \\\\
+& \text{for each kernel excess}, k \\\\
+& \textit{offset }\text{is the total kernel offset} \\\\
+& \text{and }E_l\text{ is total emission up to block } l\\\\
+\end{align}
+\tag{1}
+$$
+
+
```

Comment:
```suggestion
To include burnt outputs in the balance, we can immediately mark burnt outputs as spent (and thus allow them to be pruned), and introduce the additional requirement of tracking the total sum of all burnt output commitments.

This sum is easily verified by summing all `BURNED_KERNEL` commitments in the blockchain history. These are available to all nodes since the kernels are never pruned.

The balance equation then becomes
```